vars/Amazon.yml

```diff
@@ -5,8 +5,16 @@ python2_package_names:
   # This ensures that Ansible can install packages using python2
   - python2-rpm
 
+# The Python 2 development package names
```

Comment:
This applies for all of the `vars/` files but should variables be done alphabetically in these as well? In that case the development package variables should come before the regular package variables.

Reply:
Please see commit c1753ec.